src/Message/HostedCheckoutPurchaseResponse.php

```diff
@@ -2,13 +2,13 @@
 
 namespace Omnipay\BlueSnap\Message;
 
-use Omnipay\Common\Message\RedirectResponseInterface;
 use Omnipay\BlueSnap\Constants;
+use Omnipay\Common\Message\RedirectResponseInterface;
 
 /**
  * Response object for the HostedCheckoutPurchaseRequest. See that file for details.
  */
-class HostedCheckoutPurchaseResponse extends Response implements RedirectResponseInterface
+class HostedCheckoutPurchaseResponse extends AbstractResponse implements RedirectResponseInterface
```

Comment:
This class doesn't need everything in `Response` because non of those methods apply anyway.